READMEs/README.es-ES.md

```
@@ -0,0 +1,1370 @@
+
+```bash
+npx @agentmemory/agentmemory upgrade
+```
+
+Aviso: este comando muta el workspace/runtime actual. Puede actualizar dependencias de JavaScript, puede ejecutar `cargo install iii-engine --force`, y puede traer imágenes Docker.
+
+Los detalles de implementación están en `src/cli.ts` (ver `runUpgrade` en torno a la región `src/cli.ts:544-595`).
+
+### Claude Code (un bloque, pégalo)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)